airflow/serialization/json_schema.py

```diff
@@ -19,14 +19,15 @@
 """jsonschema for validating serialized DAG and operator."""
 
 import pkgutil
-from typing import Iterable
-
-import jsonschema
+from typing import TYPE_CHECKING, Iterable
 
 from airflow.exceptions import AirflowException
 from airflow.settings import json
 from airflow.typing_compat import Protocol
 
+if TYPE_CHECKING:
+    import jsonschema
```

Comment:
Does this change much?

Reply:
It was high up in the list of slow imports!

Reply:
A schema validation library being slow… I guess that’s how third-party libraries work.